### PR TITLE
Run unit + integration tests on travis

### PR DESCRIPTION
**Pull request by parente**
Really want integration + unit to run

### CODE REVIEW

**Comment by parente** [comment]
Runs afoul of the `make` bug that inserts garbage for `?=` overridden variables. Doh!

**Comment by parente** [comment]
This now works but required changes. Takes roughly 10 minutes.

File: Makefile [hunk 1]
```diff
@@ -132,21 +136,13 @@ logging-container: demo-container ## Run dashboard server with node debugger in
...
-		$(DOCKER_OPTIONS) \
-		$(TEST_IMAGE_NAME) $(CMD)
-
-test: | build test-container ## Run unit tests
-
-IT_SERVER_NAME:=integration-test-server
+test: ## Run unit tests
+	docker run -it --rm \
+		--name $(TEST_CONTAINER_NAME) \
+		$(TEST_IMAGE_NAME) test
```
Comment (jhpedemonte) [inline Makefile]: Missing `@`.

Also, I get `_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.` as output, but everything seems to work fine.

File: Makefile [hunk 2]
```diff
@@ -157,45 +153,41 @@ $(DASHBOARD_SERVER) -d \
...
-		-e KERNEL_GATEWAY_URL=http://$(KG_CONTAINER_NAME):8888 \
-		-e KG_AUTH_TOKEN=$(KG_AUTH_TOKEN) \
-		-e AUTH_TOKEN=$(AUTH_TOKEN) \
-		-e TEST_USERNAME=$(USERNAME) \
-		-e TEST_PASSWORD=$(PASSWORD) \
-		--link $(DASHBOARD_CONTAINER_NAME):$(DASHBOARD_CONTAINER_NAME) \
-		--link $(KG_CONTAINER_NAME):$(KG_CONTAINER_NAME)";
-@$(MAKE) kill
+@echo '-- Starting test container'
+docker run -it --rm \
```
Comment (jhpedemonte) [inline Makefile]: Missing `@`

**Comment by jhpedemonte** [comment]
Can't test Travis, but building locally still works fine. Fix missing `@` and it's good to merge.

File: Makefile (hunk 1 above)
Comment (parente) [inline Makefile]: Added @.

I see the same X11 error but it has no impact.

File: Makefile (hunk 2 above)
Comment (parente) [inline Makefile]: Fixed.

**Pull request merged**